modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library_settings/size_limit.jsp

```diff
@@ -62,6 +62,16 @@ DLSizeLimitConfigurationDisplayContext dlSizeLimitConfigurationDisplayContext =
 			<aui:input label="size-limit-copy-files" name="maxSizeToCopy" type="number" value="<%= dlSizeLimitConfigurationDisplayContext.getMaxSizeToCopy() %>" />
 		</clay:sheet-section>
 
+		<clay:sheet-section>
+			<h3 class="c-mb-2 sheet-subtitle text-secondary"><liferay-ui:message key="buffer-title" /></h3>
```

Comment:
@vamshikrishnamaddi, key should be in any case buffer-limit-title. Still better go with the variable: we either name the configuration maxBufferSize or maxBufferLimit. And change keys accordingly